core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/hybrid/HybridRoleManager.java

```diff
@@ -674,13 +674,13 @@ public Map<String, List<String>> getHybridRoleListOfUsers(List<String> userNames
             }
         }
 
-        sqlStmt = sqlStmt.replaceFirst("\\?", Matcher.quoteReplacement(usernameParameter.toString()));
         try (Connection connection = DatabaseUtil.getDBConnection(dataSource);
                 PreparedStatement prepStmt = connection.prepareStatement(sqlStmt)) {
-            prepStmt.setInt(1, tenantId);
```

Comment:
Will the removal of tenant Id from the query cause issues in multi-tenanted environment ?

Reply:
It hasn't been removed from the query. Here I am setting the username first and then the tenant Id. I have only incremented Indices, nothing is removed.